### PR TITLE
Adding sudo command as optional

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: hchiramm hchiramm@redhat.com

### CODE REVIEW

**Comment by humblec** [comment]
@lpabon can you please review this PR ?

File: ssh/ssh.go [hunk 1]
```diff
@@ -105,7 +106,7 @@ func NewSshExecWithKeyFile(logger *utils.Logger, user string, file string) *SshE
 }
 
 // This function was based from https://github.com/coreos/etcd-manager/blob/master/main.go
-func (s *SshExec) ConnectAndExec(host string, commands []string, timeoutMinutes int) ([]string, error) {
+func (s *SshExec) ConnectAndExec(host string, commands []string, timeoutMinutes int, sudoCom string) ([]string, error) {
```
Comment (lpabon) [inline ssh/ssh.go]: Instead use `useSudo bool`

File: ssh/ssh.go [hunk 2]
```diff
@@ -133,7 +134,7 @@ func (s *SshExec) ConnectAndExec(host string, commands []string, timeoutMinutes
 		session.Stdout = &b
 		session.Stderr = &berr
 
-		err = session.Start(command)
+		err = session.Start(sudoCom + " " + command)
```
Comment (lpabon) [inline ssh/ssh.go]: This would be changed so that right before line 136:

``` go
if useSudo {
    command = "sudo " + command
}
```

**Comment by humblec** [comment]
@lpabon done. thanks !

**Pull request merged**